config/runtime.exs

```diff
@@ -144,6 +144,23 @@ end
 if config_env() == :prod do
   config :dotcom, alerts_bus_stop_change_bucket: System.get_env("S3_PREFIX_BUSCHANGE")
 
+  # Extract the host fron the sentry dsn
+  sentry_dsn_host =
+    case Regex.run(~r/@(.*)\//, System.get_env("SENTRY_DSN", ""), capture: :all_but_first) do
+      nil -> ""
+      [match | _] -> match
+    end
+
+  config :dotcom,
+         :content_security_policy_definition,
+         DotcomWeb.Plug.SecureHeaders.build_csp_directives(
+           img: "#{System.get_env("STATIC_HOST", "")} #{System.get_env("CMS_API_BASE_URL", "")}",
+           style: "#{System.get_env("STATIC_HOST", "")}",
+           script: "#{System.get_env("STATIC_HOST", "")}",
+           font: "#{System.get_env("STATIC_HOST", "")}",
+           connect: "wss://#{host} #{sentry_dsn_host || ""}"
```

Comment:
This would probably be easier if you make @base_img_directives a list and then export it via base_image_directives/0 and then make a function that just takes a keyword list of lists of strings:

[
  img: base_image_directives() ++ [System.get_env("STATIC_HOST", ""), System.get_env("CMS_API_BASE_URL", "")],
  ...
]